src/ArmTemplates/Extractor/EntityExtractors/BackendExtractor.cs

```diff
@@ -161,13 +157,9 @@ void SaveBackendApiParametersToCache()
 
         bool DoesPolicyReferenceBackend(
             string policyContent,
-            IEnumerable<NamedValueTemplateResource> namedValueResources,
             string backendName,
             BackendTemplateResource backendTemplateResource)
         {
-            // a policy is referenced by a backend with the set-backend-service policy, which will reference use the backends name or url, or through referencing a named value that applies to the backend
-            var namedValueResourcesUsedByBackend = namedValueResources.Where(resource => this.DoesBackendReferenceNamedValue(resource, backendTemplateResource));
-            
             if (backendName != null && policyContent.Contains(backendName) ||
```

Comment:
```suggestion
return (!string.IsNullOrEmpty(backendName) && policyContent.Contains(backendName) ||
backendTemplateResource.Properties.Url != null && policyContent.Contains(backendTemplateResource.Properties.Url) ||
                backendTemplateResource.Properties.Title != null && policyContent.Contains(backendTemplateResource.Properties.Title) ||
                backendTemplateResource.Properties.ResourceId != null && policyContent.Contains(backendTemplateResource.Properties.ResourceId))
```

Reply:
updated all checks in this code block, thanks for review!